tests/run/iarrays.scala

```diff
@@ -0,0 +1,53 @@
+import reflect.ClassTag
+object Test extends App {
+
+  val xs = IArray(1, 2, 3)
+
+  def f[T](ys: IArray[T]) = {
```

Comment:
What type does this method erase to? `f(ys: Array[AnyRef])`?

Reply:
No, it's `Object`. Just what `Array[T]` erases to.